File: ITServiceDesk.Core/Entities/TicketCategory.cs
namespace ITServiceDesk.Core.Entities;

public class TicketCategory : BaseEntity
{
    public string Name { get; set; } = string.Empty;
    public string? Description { get; set; }
    
    // Navigation Properties
    public ICollection<Ticket> Tickets { get; set; } = [];
}

File: ITServiceDesk.Client/src/types/ticketCategory.ts
export interface TicketCategoryDto {
    id: string;
    name: string;
    description?: string;
}

export interface TicketCategoryCreateDto {
    name: string;
    description?: string;
}

export interface TicketCategoryUpdateDto {
    id: string;
    name: string;
    description?: string;
}
